packages/dataviews-client/src/resources/resources-resolvers.ts

```diff
@@ -6,6 +6,7 @@ import {
   UrlDataviewInstance,
 } from '../resolve-dataviews'
 
+// TODO Deprecate
```

Comment:
Aren't we removing this because it is already in use for vessel-examiner?

Reply:
See https://github.com/GlobalFishingWatch/frontend/pull/820